src/mobile-app/components/uploader.tsx

```diff
@@ -157,13 +157,32 @@ export const Uploader = (props: IProps) => {
       <div className={css.dialog}>
         <div className={css.header}>
           Upload Experiment
-          <div className={css.floatRight}>
-            <span onClick={props.onClose}>{uploadState === UploadState.Uploaded ? "OK" : "Cancel"}</span>
-          </div>
+
         </div>
         <div className={css.contents}>
-          <RunInfoComponent run={props.run} />
+          <div className={css.run}>
+            <div className={css.initials}><Initials text={props.run.experiment.metadata.initials}/></div>
+            <div className={css.text}>
+              <RunInfoComponent run={props.run} />
+            </div>
+          </div>
+          <div className={css.instructions}>
+            <ol>
+              <li>In the [WATERS] activity, click the Import button to display a QR code.</li>
+              <li>Point your camera at the QR code to begin uploading this experiment.</li>
+            </ol>
+          </div>
+          <div className={css.resultText}>
+            <div className={css.scanInfo}>
+              Position QR code within guides to scan
```

Comment:
should we also add "and then click the screen to scan it"?

Reply:
Good call - I've made adjustments to the text in both cases.